src/request.rs

```diff
@@ -44,7 +44,7 @@ pub trait KvRequest: Sync + Send + 'static + Sized {
         mut self,
         pd_client: Arc<impl PdClient>,
     ) -> BoxStream<'static, Result<Self::RpcResponse>> {
-        let stores = self.store_stream(pd_client.clone());
+        let stores = self.store_stream(pd_client.clone(), false);
```

Comment:
I think it's not correct to always set it to false. Parameter `is_retry` should be also added to this method and set `is_retry` to `true` on region error.